aws/resource_aws_ecs_cluster.go

```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/terraform-providers/terraform-provider-aws/aws/internal/keyvaluetags"
 )
 
+const (
+	timeoutCreate = 10 * time.Minute
+	timeoutDelete = 10 * time.Minute
+	timeoutUpdate = 10 * time.Minute
```

Comment:
Nit: These are global constants across all resources and should likely be more explicitly named:

```go
	ecsClusterTimeoutCreate = 10 * time.Minute
	ecsClusterTimeoutDelete = 10 * time.Minute
	ecsClusterTimeoutUpdate = 10 * time.Minute
```